### PR TITLE
changed the colors, and the size

### PR DESCRIPTION
**Pull request by IsabelleHoffmann**

### CODE REVIEW

File: MoveTheSquare/src/edu/ycp/cs320/movethesquare/ui/GameView.java [hunk 1]
```diff
@@ -17,16 +17,16 @@
...
-	private static final Color MIDNIGHT_BLUE = new Color(25, 25, 112);
+	private static final Color DARK_GREEN = new Color(1, 50, 32);
 	
 	private Game model;
 	private GameController controller;
 	private Timer timer;
 	
 	public GameView(Game model) {
 		this.model = model;
-		setPreferredSize(new Dimension((int) model.getWidth(), (int)model.getHeight()));
```
Comment (IsabelleHoffmann) [inline MoveTheSquare/src/edu/ycp/cs320/movethesquare/ui/GameView.java:29]: want this

File: MoveTheSquare/src/edu/ycp/cs320/movethesquare/ui/GameView.java [hunk 2]
```diff
@@ -17,16 +17,16 @@
 import edu.ycp.cs320.movethesquare.model.Square;
 
 public class GameView extends JPanel {
-	private static final Color MIDNIGHT_BLUE = new Color(25, 25, 112);
+	private static final Color DARK_GREEN = new Color(1, 50, 32);
```
Comment (csims2) [inline MoveTheSquare/src/edu/ycp/cs320/movethesquare/ui/GameView.java]: delete

File: MoveTheSquare/src/edu/ycp/cs320/movethesquare/ui/GameView.java [hunk 3]
```diff
@@ -17,16 +17,16 @@
...
 	private Game model;
 	private GameController controller;
 	private Timer timer;
 	
 	public GameView(Game model) {
 		this.model = model;
-		setPreferredSize(new Dimension((int) model.getWidth(), (int)model.getHeight()));
-		setBackground(MIDNIGHT_BLUE);
+		setPreferredSize(new Dimension((int) model.getWidth()-5, (int)model.getHeight()-5));
+		setBackground(DARK_GREEN);
```
Comment (csims2) [inline MoveTheSquare/src/edu/ycp/cs320/movethesquare/ui/GameView.java:34]: delete

File: MoveTheSquare/src/edu/ycp/cs320/movethesquare/ui/GameView.java [hunk 4]
```diff
@@ -66,7 +66,7 @@ protected void paintComponent(Graphics g) {
 		
 		// djh2-KEC110-21: changed from GREEN to RED
 		// djh2-YCPlaptop: change from RED to YELLOW
-		g.setColor(Color.YELLOW);
```
Comment (csims2) [inline MoveTheSquare/src/edu/ycp/cs320/movethesquare/ui/GameView.java:70]: delete

**Review by csims2 — APPROVED** [review]

**Pull request merged**